### ISSUE TITLE
Enderman Spawn Egg spawns Ender Dragon

### ISSUE THREAD

**Issue by Siggiio** [GeyserMC/Geyser#3695]
### Describe the bug

When you take an Enderman Spawn Egg from the creative inventory on Bedrock Edition, it spawns an *Ender Dragon* Spawn Egg, but looks like an *Enderman* Spawn Egg.

### To Reproduce

1. Take an **Enderman Spawn Egg** from the creative inventory
2. Place it and an ***Ender Dragon*** will appear.
3. Drop the item and have someone pick it up on Java Edition, and they will get an ***Ender Dragon* Spawn Egg**

### Expected behaviour

Taking an Enderman Spawn Egg from the creative inventory should not result in it becoming an Ender Dragon Spawn Egg and looking like an Enderman Spawn Egg.

### Screenshots / Videos

Video showing PS4 and Java:
https://www.youtube.com/watch?v=Zt63dlkBF1g

### Server Version and Plugins

git-Paper-519 (MC: 1.19.4) (Implementing API version 1.19.4-R0.1-SNAPSHOT) (Git: 936205b)

Plugins:
floodgate, Geyser-Spigot

### Geyser Dump

https://dump.geysermc.org/qUe2fRZNz1kjdB9QiUdRYjMT5JhRg75R

### Geyser Version

2.1.0-SNAPSHOT (git-master-0f99abc)

### Minecraft: Bedrock Edition Device/Version

1.19.73 PlayStation 4, iOS

**Comment by onebeastchris** [comment]
Duplicate of https://github.com/GeyserMC/Geyser/issues/3675 - will also be fixed once https://github.com/GeyserMC/Geyser/pull/3688 is merged

**Comment by Chew** [comment]
Closing as stated above. Thank you for reporting!
